rootex/framework/entity.cpp

```diff
@@ -49,6 +51,10 @@ JSON::json Entity::getJSON() const
 	{
 		j["Components"][component->getName()] = component->getJSON();
 	}
+	if (m_Script)
+	{
+		j["Entity"]["script"] = m_Script->getJSON();
+	}
```

Comment:
Avoid conditionally adding data in the JSON. Send in `nullptr` in the `script` field if no script exists. This will be useful for debugging